app/components/v2/live_release/pre_prod_release/submission_component.rb

```diff
@@ -53,6 +49,10 @@ def last_activity_tooltip
     "Last activity at #{time_format(last_activity_ts)}"
   end
 
+  def last_activity_tick?
+    %w[preprocessing preparing created].include?(submission.status)
```

Comment:
do we need it for `created`? it will be in that state only if auto-trigger if false.